----------START---------

HTTPConnectionPool(host='meteo.arso.gov.si', port=80): Max retries exceeded with url: /uploads/probase/www/observ/surface/text/sl/observationAms_ZADLOG_history.html (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fbc03f6d1d0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',))
----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

HTTPConnectionPool(host='meteo.arso.gov.si', port=80): Max retries exceeded with url: /uploads/probase/www/observ/surface/text/sl/observationAms_ZADLOG_history.html (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7faf4a5eb350>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',))
----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

HTTPConnectionPool(host='meteo.arso.gov.si', port=80): Max retries exceeded with url: /uploads/probase/www/observ/surface/text/sl/observationAms_ZADLOG_history.html (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f2ffb83a350>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',))
----------START---------

HTTPConnectionPool(host='meteo.arso.gov.si', port=80): Max retries exceeded with url: /uploads/probase/www/observ/surface/text/sl/observationAms_ZADLOG_history.html (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f06940eb350>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',))
----------START---------

-----------END-----------

----------START---------

-----------END-----------

----------START---------

HTTPConnectionPool(host='meteo.arso.gov.si', port=80): Max retries exceeded with url: /uploads/probase/www/observ/surface/text/sl/observationAms_ZADLOG_history.html (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f51cdc32290>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution',))
